pybamm/models/submodels/electrode/ohm/full_ohm.py

```diff
@@ -3,22 +3,13 @@
 #
 import pybamm
 from .base_ohm import BaseModel
+from pybamm.doc_utils import parameters_from, extends
 
 
+@parameters_from
+@extends
```

Comment:
can you rename to `copy_parameter_doc_from_parent` and `doc_extend_parent`

Reply:
Yes